### PR TITLE
More test graphs and README

### PR DESCRIPTION
**Pull request by wilmhit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Elegant graph coloring (labeling).
+
+## Running
+
+Install python then:
```
Comment (songofroland) [inline README.md:5]: then?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+G. J. Chang, D. F. Hsu, and D. G. Rogers, Additive variations on a graceful theme:
+some results on harmonious and other related graphs, Congr. Numer., 32 (1981)
+181-197
+
+Graph Labeling
+Joseph A. Gallian
+2018-12-15
+116-118
+Available [here](https://www.combinatorics.org/ojs/index.php).
```
Comment (songofroland) [inline README.md:32]: never saw that

File: graphs/CHAIN_UNSAT.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+0 1 0 0 0 0 0
```
Comment (songofroland) [inline graphs/CHAIN_UNSAT.txt:1]: It's a new one?

File: tests/test_graph_coloring.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 from graph_coloring.custom_types import AdjMatrix
 from graph_coloring.coloring import elegantly_color_edges
-from graph_coloring.sample_graphs import SHACK_F_4, DI_3, STAR_S_6
+from .sample_graphs import SHACK_F_4, DI_3, STAR_S_6
```
Comment (songofroland) [inline tests/test_graph_coloring.py:5]: Ideally tt should be either relative or absolute

**Review by songofroland — APPROVED** [review]
Well done

File: README.md (hunk 1 above)
Comment (wilmhit) [reply README.md:5]: Then do steps below?

File: README.md (hunk 2 above)
Comment (wilmhit) [reply README.md:32]: Yes you did. It's first document we ever read.

File: graphs/CHAIN_UNSAT.txt (hunk 3 above)
Comment (wilmhit) [reply graphs/CHAIN_UNSAT.txt:1]: Yes

File: README.md (hunk 1 above)
Comment (songofroland) [reply README.md:5]: _Install python, then:_

File: tests/test_graph_coloring.py (hunk 4 above)
Comment (wilmhit) [reply tests/test_graph_coloring.py:5]: What do you mean. It is relative to package.

Comment (songofroland) [reply tests/test_graph_coloring.py:5]: it should go always either
`from graph_coloring.sample_graphs import X`
or
`from .sample_graphs import Y`

mixing is a bad practice

Comment (wilmhit) [reply tests/test_graph_coloring.py:5]: But these are 2 different packages

File: graphs/CHAIN_UNSAT.txt (hunk 3 above)
Comment (songofroland) [reply graphs/CHAIN_UNSAT.txt:1]: It's unsatisfiable :|

Comment (songofroland) [reply graphs/CHAIN_UNSAT.txt:1]: Screenshot 2021-06-01 at 15 50 02

File: tests/test_graph_coloring.py (hunk 4 above)
Comment (songofroland) [reply tests/test_graph_coloring.py:5]: then okay

**Pull request merged**